src/deployment/deploy.py

```diff
@@ -1238,7 +1280,14 @@ def main() -> None:
         )
         states = rbac_only_states
     else:
-        states = full_deployment_states
+        if args.dotnet_deploy:
+            logger.info(
+                "deploying dotnet services for Azure functions instead of python"
+            )
```

Comment:
The option here should be to deploy both C# and python or just python. 
We will need to have those functions running side by side

Reply:
Gotcha, the default being just building python then, correct?

I'll adjust the --dotnet_deploy flag for building both C# & python if so.

Reply:
"deploy" instead of "build". But yeah that's the idea